cfgov/unprocessed/css/print.less

```
@@ -38,10 +38,20 @@
         content: " (" attr(href) ")";
     }
 
-    a[href^="/"]:not([href*="?authors"]):not([href="/"]):after {
-        // Output root-relative hrefs in parentheses with our domain prepended,
-        // as long as they are not author links or the logo homepage link.
-        content: " (https://www.consumerfinance.gov" attr(href) ")";
+    a[href^="/"]:after {
+        // Output root-relative hrefs in parentheses with our domain prepended.
+        content: " (https://www.cfpb.gov" attr(href) ")";
```

Might as well go whole-hog and ditch the `www.`, right?

```suggestion
        content: " (https://cfpb.gov" attr(href) ")";
```

---

Works for me. I bet we put the www for either A) branding reasons or B) because non-www doesn't resolve on our internal network. For printing, keeping it as short as possible makes sense.

---

I'd vote to keep the `www` and drop the `https://`

---

Why not both? 😆 

The shorter the better! Let's drop everything that's not necessary for a user to get there.

---

Oooh I'll drop both and request feedback in stand-up.